docs/platform/developing/flashing-rpi.md

```diff
@@ -451,20 +278,14 @@ To configure the Raspberry Pi board, follow the steps below:
 1.  Connect Smart Development Bridge (SDB) as described in [Connect the board to the PC](#connect-the-board-to-the-pc).
 2.  Install the connectivity drivers for each board:
     -   **Raspberry Pi 3/4**
-        1.  Download the plugin zip file from the **Raspberry Pi 3/4(6.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
+        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware> and follow the instructions.
```

Comment:
```suggestion
        1.  Download the plugin zip file from the **Raspberry Pi 3/4(7.0) Plugin** section at <http://developer.samsung.com/tizendevice/firmware>{:target="_blank"} and follow the instructions.
```

Reply:
External links in a .md file opens in a new window

Reply:
Updated. Thanks!